samples/subsys/mgmt/osdp/prj.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_PRINTK=y
+CONFIG_LOG=y
+
+# OSDP config
+CONFIG_OSDP=y
+CONFIG_OSDP_UART_DEV_NAME="UART_2"
```

Comment:
Should this be using a Devicetree `chosen` property instead? @galak

Reply:
Yes, that has been the convention we've be using in general.

Reply:
How should I handle the case where board doesn't define an osdp node? Can I fallback to this method?

Reply:
Look at something like subsys/bluetooth/common/Kconfig and `BT_MONITOR_ON_DEV_NAME` & `zephyr,bt-mon-uart` for example.

Reply:
Done.